File: abhishek-files/countdown_Timer.py
# Your code here

def countdown(start_num):
    # Ensure input is positive
    if start_num <= 0:
        print("Please enter a positive integer.")
        return
    
    while start_num > 0:
        print(start_num)
        start_num -= 1  # Decrease by 1 each loop
    print("Blast off!")

# --- Test Case ---
print("Starting 5-second countdown:")
countdown(5)

# Expected Output:
# 5
# 4
# 3
# 2
# 1
# Blast off!

File: harshal-file/CountdownTimer.java
public class CountdownTimer {

    public static void main(String[] args) {
        // --- Test Case ---
        System.out.println("Starting 5-second countdown:");
        countdown(5);
        // Expected Output:
        // 5
        // 4
        // 3
        // 2
        // 1
        // Blast off!
    }

    // Your method here
    public static void countdown(int startNum) {
       for(int i=startNum;i>=0;i--)
       {
        System.out.println(i);
       }
    }
}
